mmcv/utils/config.py

```diff
@@ -638,6 +638,8 @@ def _parse_int_float_bool(val):
             pass
         if val.lower() in ['true', 'false']:
             return True if val.lower() == 'true' else False
+        if val.lower() == 'none':
+            return None
```

Comment:
suggest moving those lines to line 693

Reply:
Contextually it belongs here, because the code around 693 handles the different delimiters until it finds a single value, then this function handles the rest. 
Probably the name is not accurate any more, because it should be `_parse_int_float_bool_none` now.

Reply:
Agree to rename it to `_parse_int_float_bool_none`.

Reply:
Sometimes string "none" is an available option, e.g., in losses like below
https://github.com/open-mmlab/mmdetection/blob/3e2693151add9b5d6db99b944da020cba837266b/mmdet/models/losses/iou_loss.py#L250

When setting `reduction="none"` in loss, will line 641 change it from 'none' to None? This might cause failure.

Reply:
That's a good point. However, this only has an impact if it's passed in from the command line, e.g. `--cfg-options loss.reduction=none`.

But to cover that case, I think either the parser should be refactored to only parse inputs as string if they're quoted, which might be a rather complex change and would lead to confusion, or just force the parser to only accept `None` as python `None` and `none` and string `"none"`, but then it's inconsistent with the bool case.

Reply:
Yes, we could force the parser to only accept None as python None. We do not need to make it consistent with the bool case.

Reply:
Done